bindings/c/test/mako/mako.c

```diff
@@ -1888,7 +1929,7 @@ int validate_args(mako_args_t* args) {
 #define STATS_TITLE_WIDTH 12
 #define STATS_FIELD_WIDTH 12
 
-void print_stats(mako_args_t* args, mako_stats_t* stats, struct timespec* now, struct timespec* prev) {
+void print_stats(mako_args_t* args, mako_stats_t* stats, struct timespec* now, struct timespec* prev, FILE* fp) {
```

Comment:
fp means function pointer in my mind. fd, file descriptor might be a better choice

Reply:
`fp` is a very common name for FILE pointer

Reply:
Works for me then